solarkraft/test/unit/verify.instrumentedMonitor.ts

```diff
@@ -19,24 +19,37 @@ const instrumentedMonitor = {
                         args: [
                             {
                                 kind: 'OperEx',
-                                oper: 'EQ',
+                                oper: 'AND',
                                 type: 'Untyped',
                                 args: [
                                     {
-                                        kind: 'NameEx',
-                                        name: 'is_initialized',
-                                        type: 'Untyped',
-                                    },
-                                    {
-                                        kind: 'ValEx',
+                                        kind: 'OperEx',
+                                        oper: 'EQ',
                                         type: 'Untyped',
-                                        value: {
-                                            kind: 'TlaBool',
-                                            value: false,
-                                        },
+                                        args: [
+                                            {
+                                                kind: 'NameEx',
+                                                name: 'is_initialized',
+                                                type: 'Untyped',
+                                            },
+                                            {
+                                                kind: 'ValEx',
+                                                type: 'Untyped',
+                                                value: {
+                                                    kind: 'TlaBool',
+                                                    value: false,
+                                                },
+                                            },
+                                        ],
                                     },
                                 ],
                             },
+                            {
+                                kind: 'OperEx',
+                                oper: 'AND',
+                                type: 'Untyped',
+                                args: [],
+                            },
```

Comment:
Ah interesting, I had not considered before that empty arrays will generate empty conjunctions.

It's correct ofc and inconsequential for now, but I'm kinda scared this will produce a mean bug if we ever generate disjunctions in the future 😅

Reply:
That's a reasonable concern, do you think it'd be better if I merged it so that instead of 
```
and(
  and(present1, ..., presentN),
  and(missing1, ..., missingM)
)
```
it's just a flat
```
and(present1, ..., presentN, missing1, ..., missingM)
```
? It shouldn't be too complicated, since you're just merging two argument arrays.

Reply:
Yeah, I thought about it, but I'm undecided...
In the end, it's doing what it's supposed to do, and `tlaAnd` does imply what happens if you pass an empty array. I think we can keep it as-is.

Reply:
👍